telegram/ext/_commandhandler.py

```diff
@@ -216,78 +237,84 @@ class PrefixHandler(CommandHandler):
             Defaults to :obj:`False`.
 
     Attributes:
+        prefixes (Set[:obj:`str`]):
+            The prefix(es) that will precede :attr:`commands`.
+        commands (Set[:obj:`str`]):
+            The set of command(s) that this handler should listen for.
         callback (:obj:`callable`): The callback function for this handler.
         filters (:class:`telegram.ext.BaseFilter`): Optional. Only allow updates with these
             Filters.
         run_async (:obj:`bool`): Determines whether the callback will run asynchronously.
 
+            .. versionchanged:: 14.0
+                :attr:`prefix` has been changed to :attr:`prefixes` and accepts sets as well.
+                :attr:`command` has been changed to :attr:`commands` and accepts sets as well.
     """
 
-    # 'prefix' is a class property, & 'command' is included in the superclass, so they're left out.
-    __slots__ = ('_prefix', '_command', '_commands')
+    __slots__ = ('_prefixes', '_combinations')
 
     def __init__(
         self,
-        prefix: SLT[str],
-        command: SLT[str],
+        prefix: SLTS[str],
+        command: SLTS[str],
         callback: Callable[[Update, CCT], RT],
         filters: filters_module.BaseFilter = None,
         run_async: Union[bool, DefaultValue] = DEFAULT_FALSE,
     ):
 
-        self._prefix: List[str] = []
-        self._command: List[str] = []
-        self._commands: List[str] = []
-
         super().__init__(
-            'nocommand',
+            command,
             callback,
             filters=filters,
             run_async=run_async,
         )
 
-        self.prefix = prefix  # type: ignore[assignment]
-        self.command = command  # type: ignore[assignment]
+        self._prefixes: Set[str] = set()
+        self._combinations: Set[str] = set()
+
+        self.prefixes = prefix  # type: ignore[assignment]
         self._build_commands()
 
     @property
-    def prefix(self) -> List[str]:
+    def prefixes(self) -> Set[str]:
         """
         The prefixes that will precede :attr:`command`.
 
         Returns:
-            List[:obj:`str`]
+            Set[:obj:`str`]
         """
-        return self._prefix
+        return self._prefixes
 
-    @prefix.setter
-    def prefix(self, prefix: Union[str, List[str]]) -> None:
+    @prefixes.setter
+    def prefixes(self, prefix: SLTS) -> None:
         if isinstance(prefix, str):
-            self._prefix = [prefix.lower()]
+            self._prefixes = {prefix.lower()}
         else:
-            self._prefix = prefix
+            self._prefixes = set(prefix)
         self._build_commands()
 
-    @property  # type: ignore[override]
-    def command(self) -> List[str]:  # type: ignore[override]
+    @property
+    def commands(self) -> Set[str]:
```

Comment:
Is this override still necessary? IISC, this does exatly the same as `CommandHandler.commands`

Reply:
yes, because this class uses a different setter than the parent one and needs a property to set it.

Reply:
IIRC using `@CommandHandler.commands.setter` instead of `@commands.setter` should do the trick and then you don't need to re-implement the property